jolly-jellyfish/src/page_maker/templates/page_maker/webpage_list.html

```diff
@@ -0,0 +1,16 @@
+{% extends "page_maker/base.html" %}
+
+{% block content %}
+{% for webpage in webpages %}
```

Comment:
```suggestion
{% block content %}
<h2>Top Pages</h2>
{% for webpage in webpages %}
```
As above, styling concerns.